twitter-processor-example/activiti-cloud-connectors-3rd-party/src/main/java/org/activiti/cloud/connectors/external/FilterController.java

```diff
@@ -7,7 +7,7 @@
 @RestController
 public class FilterController {
 
-    public static String currentFilter = "TRUMP";
+    public static String currentFilter = "OXFORD";
```

Comment:
What about loading the default value from a properties file? Will it be a future step?